README.md

```diff
@@ -1,71 +1,138 @@
-
-The key features of Nomad are:
-
-* **Docker Support**: Jobs can specify tasks which are Docker containers.
-  Nomad will automatically run the containers on clients which have Docker
-  installed, scale up and down based on the number of instances requested, and
-  automatically recover from failures.
-
-* **Operationally Simple**: Nomad runs as a single binary that can be
-  either a client or server, and is completely self contained. Nomad does
-  not require any external services for storage or coordination. This means
-  Nomad combines the features of a resource manager and scheduler in a single
-  system.
-
-* **Multi-Datacenter and Multi-Region Aware**: Nomad is designed to be
-  a global-scale scheduler. Multiple datacenters can be managed as part
-  of a larger region, and jobs can be scheduled across datacenters if
-  requested. Multiple regions join together and federate jobs making it
-  easy to run jobs anywhere.
-
-* **Flexible Workloads**: Nomad has extensible support for task drivers, allowing it to run
-  containerized, virtualized, and standalone applications. Users can easily start Docker
-  containers, VMs, or application runtimes like Java. Nomad supports Linux, Windows, BSD, and OSX,
-  providing the flexibility to run any workload.
-
-* **Built for Scale**: Nomad was designed from the ground up to support global scale
-  infrastructure. Nomad is distributed and highly available, using both
-  leader election and state replication to provide availability in the face
-  of failures. Nomad is optimistically concurrent, enabling all servers to participate
-  in scheduling decisions which increases the total throughput and reduces latency
-  to support demanding workloads. Nomad has been proven to scale to cluster sizes that
-  exceed 10k nodes in real-world production environments.
-
-* **HashiCorp Ecosystem**: HashiCorp Ecosystem: Nomad integrates with the 
-entire HashiCorp ecosystem of tools. Like all HashiCorp tools, Nomad follows 
-the UNIX design philosophy of doing something specific and doing it well. 
-Nomad integrates with Terraform, Consul, and Vault for provisioning, service 
-discovery, and secrets management.
-
-For more information, see the [introduction section](https://www.nomadproject.io/intro)
-of the Nomad website.
-
-Getting Started & Documentation
+Overview
+-------------------------------
+
+Nomad is a flexible, lightweight, easy-to-use, performant workload orchestrator that deploys:
+
+* [Containers](https://www.nomadproject.io/docs/drivers/docker.html)
+* [Legacy applications](https://www.nomadproject.io/docs/drivers/exec.html)
+* [Virtual machines](https://www.nomadproject.io/docs/drivers/qemu.html)
+
+Nomad enables developers to use declarative infrastructure-as-code for deploying their applications (jobs).  Nomad uses bin packing to efficiently schedule jobs and optimize for resource utilization.  Nomad is supported on MacOS, Windows, and Linux.
```

Comment:
```suggestion
Nomad enables developers to use declarative infrastructure-as-code for deploying their applications (jobs).  Nomad uses bin packing to efficiently schedule jobs and optimize for resource utilization.  Nomad is supported on macOS, Windows, and Linux.
```

small typo fix

Reply:
Ooh, very good catch!  Thank you Dani. :D